conda-forge.yml

```diff
@@ -1,12 +1,14 @@
+bot:
+  automerge: true
 build_platform:
   osx_arm64: osx_64
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
 provider:
-  linux_aarch64: default
-  linux_ppc64le: default
+  linux_aarch64: azure
```

Comment:
What's the purpose of changing this? Is it being changed in all recipes from default to azure?

Reply:
@danielnachun You need to react here.